pages/stmn/create.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <section>
+    <br />
+    <b-tabs v-model="activeTab" destroy-on-hide expanded>
+      <b-tab-item v-for="x in components" :key="x" :label="x">
+        <component
+          :is="x"
+          v-if="components[activeTab] === x"
+          :show-explainer-text="showExplainerText"
+          @navigateToCreateNftTab="switchToNft" />
+      </b-tab-item>
+    </b-tabs>
+  </section>
+</template>
+
+<script lang="ts">
+import { Component, mixins } from 'nuxt-property-decorator'
```

Comment:
check `/pages/_prefix/collection/_id/index.vue` or `/pages/_prefix/unlockable.vue` for vue3 layout

Reply:
will do. thanks

Reply:
done